src/components/Interpretations/common/Message/MessageIconButton.js

```diff
@@ -51,6 +51,22 @@ const MessageIconButton = ({
                         color: ${colors.teal600};
                         font-weight: 500;
                     }
+
+                    .button:hover {
+                        color: ${colors.grey900};
+                    }
+
+                    .button.selected:hover {
+                        color: ${colors.teal800};
+                    }
+
+                    .button.selected :global(svg) {
+                        color: ${colors.teal500};
+                    }
+
+                    .button.selected:hover :global(svg) {
+                        color: ${colors.teal700};
```

Comment:
The svg will be `teal700` and the button (line 60) will be `teal800`. Are these intended to be different?

Reply:
Because the SVG has such thin lines, `teal800` almost looked grey, so I went for a lighter version.